src/Registration.js

```diff
@@ -0,0 +1,22 @@
+import FormInput from "./FormInput";
+import "./style/style.scss";
+
+const Registration = ({minSymbol, maxSymbol}) => {
+  const passwordRegex = new RegExp("^[-/\\=!#$%&'*+?^_`{|}~.A-Z\d]{" + minSymbol + "," + maxSymbol + "}$", "i");
+  return (
+    <div className={'modal'}>
+      <form style={{margin: '20px 0'}}>
+        <FormInput label={'Password'}
+                   validation={passwordRegex}
+                   type={'password'}
+                   required={'required'}
+                   name={'hidden'}/>
+        <FormInput label={'Confirm password'}
+                   type={'password'}
+                   required={'required'}
+                   name={'hidden'}/>
+      </form>
```

Comment:
1. form will be included into the another ticket
2. password "with open-close eyes" not used when used "password" and "confirm password" fields